src/main/java/com/appoptics/integrations/kafka/broker/Reporter.java

```diff
@@ -128,7 +128,9 @@ public void run() {
                 if (values.size() == 1) {
                     Number value = values.get("value");
                     TaggedMeasure taggedMeasure = new TaggedMeasure(new GaugeMeasure(metricName, value.doubleValue()));
-                    taggedMeasure.addTag(new Tag("source", source));
+                    for (Tag tag : this.tags) {
+                        taggedMeasure.addTag(tag);
+                    }
```

Comment:
since this is repeated twice, maybe it should be refactored into a method that takes a `TaggedMeasure` and a collection of `Tag`s and adds them all. e.g. `void addAllTags(TaggedMeasure taggedMeasure, Collection<? extends Tag> tags) { ...}`